frontend/src/components/core/Button.tsx

```diff
@@ -20,14 +21,14 @@ export const DefaultButton = styled.button`
 `;
 
 export const PrimaryButton = styled(DefaultButton)<any>`
-  font-size: ${({ theme }) => theme.fontSize.large};
-  font-weight: bold;
-  background-color: ${({ theme }) => theme.colors.blue};
+  font-size: ${({ theme }) => theme.fontSize.mediumLarge};
+  font-weight: 400;
+  background: ${({ theme }) => theme.colors.blueCircle};
   color: ${({ theme }) => theme.colors.white};
-  width: ${({ width }) => width || '32vw'};
-  height: ${({ height }) => height || '8vw'};
-  min-width: 280px;
-  min-height: 70px;
+  width: ${({ width }) => width || '16vw'};
+  height: ${({ height }) => height || '4vw'};
```

Comment:
Thanks for this - the button was definitely way too huge beforehand lol

Reply:
Haha yeah no problem, I got the styling primarily from your [Figma](https://www.figma.com/file/RqMbCSJslbRDilJScKCVib/RocketDen?node-id=0%3A1), so thanks for the design 👍